core/src/main/scala/fs2/pull1.scala

```diff
@@ -181,8 +181,26 @@ private[fs2] trait pull1 {
     if (n <= 0) Pull.pure(h)
     else Pull.awaitLimit(if (n <= Int.MaxValue) n.toInt else Int.MaxValue)(h).flatMap {
       case chunk #: h => Pull.output(chunk) >> take(n - chunk.size.toLong)(h)
+    } 
+
+  /** Emits the last `n` elements of the input. */
+  def takeRight[F[_],I](n: Long): Handle[F,I] => Pull[F,I,Handle[F,I]]  = {
```

Comment:
This should return a `Pull[F,Nothing,Vector[I]]`. There's no need to return the output `Handle` since it will always be empty!